### PR TITLE
Debezium 1.7.0.Beta1 Release Announcement

### PR DESCRIPTION
**Pull request by rk3rn3r**

### CODE REVIEW

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+date:   2021-08-25
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox ]
+author: rkerner
+---
+
+It's my pleasure to announce the first release of the Debezium 1.7 series, *1.7.0.Beta1*!
+
+This release brings https://docs.nats.io/developing-with-nats-streaming/streaming#nats-streaming-overview[NATS Streaming]
+support for Debezium Server along with many other fixes and enhancements. +
+Also this release is the first one tested with Apache Kafka 2.8.
```
Comment (jpechane) [inline _posts/2021-08-25-debezium-1-7-beta1-released.adoc:13]: Please add the mention of Kafka 2.8 into version summary in version metadata too.

Comment (rk3rn3r) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:13]: I added "kafka" to the tags now. That's what you meant?

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  Debezium 1.7.0.Beta1 Released
+date:   2021-08-25
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox, kafka ]
```
Comment (rk3rn3r) [inline _posts/2021-08-25-debezium-1-7-beta1-released.adoc:5]: here

Comment (jpechane) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:5]: @rk3rn3r I want to see it here https://debezium.io/releases/1.7/ so you should update https://github.com/debezium/debezium.github.io/blob/develop/_data/releases/1.7/1.7.0.Beta1.yml#L4 as well

File: _data/releases/1.7/1.7.0.Beta1.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 date: 2021-08-24
 version: "1.7.0.Beta1"
 stable: false
-summary: NATS support in Debezium Server
+summary: NATS support in Debezium Server; Kafka 2.8.0
```
Comment (rk3rn3r) [inline _data/releases/1.7/1.7.0.Beta1.yml:4]: like that?

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc (hunk 2 above)
Comment (rk3rn3r) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:5]: oh, I think you mean `series.yml`

File: _data/releases/1.7/1.7.0.Beta1.yml (hunk 3 above)
Comment (jpechane) [reply _data/releases/1.7/1.7.0.Beta1.yml:4]: Yes, exactly, thanks

File: _data/releases/1.7/series.yml [hunk 4]
```diff
@@ -25,7 +25,7 @@ compatibility:
         - 8.0.x
     driver:
       versions:
-        - 8.0.21
+        - 8.0.26
```
Comment (rk3rn3r) [inline _data/releases/1.7/series.yml:28]: also added this

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc (hunk 1 above)
Comment (gunnarmorling) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:13]: @rk3rn3r I assume this is what Jiri means: https://github.com/debezium/debezium.github.io/blob/develop/_data/releases/1.7/1.7.0.Beta1.yml

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc (hunk 2 above)
Comment (gunnarmorling) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:5]: It should be mentioned next to NATS as a key change in the description

Comment (gunnarmorling) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:5]: Ah, done now :)

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+author: rkerner
+---
+
+It's my pleasure to announce the first release of the Debezium 1.7 series, *1.7.0.Beta1*!
```
Comment (gunnarmorling) [inline _posts/2021-08-25-debezium-1-7-beta1-released.adoc:9]:
```suggestion
It's my pleasure to announce the second release of the Debezium 1.7 series, *1.7.0.Beta1*!
```

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+Also this release has a breaking change for the MySQL Connector. +
+The MySQL driver was updated to the latest version 8.0.26 with https://issues.jboss.org/browse/DBZ-3833[DBZ-3833].
+This update comes with a new timezone handling and configuration options. +
+Detailed information can be found in the https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-connp-props-datetime-types-processing.html[MySQL docs].
+
+This month we added the https://debezium.io/documentation/reference/operations/debezium-ui.html[Debezium UI]
+to our regular release process. If you want to learn more about the Debezium UI have a look at our
+recent link:/blog/2021/08/12/introducing-debezium-ui/[release announcement].
+
+Overall, https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12317320&version=12354171[81 issues] were fixed for this release. +
```
Comment (gunnarmorling) [inline _posts/2021-08-25-debezium-1-7-beta1-released.adoc:26]: That link points to Alpha1.

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+Thanks a lot to all contributors!
```
Comment (gunnarmorling) [inline _posts/2021-08-25-debezium-1-7-beta1-released.adoc:27]: All contributors should be listed, see this workflow here for compiling the list: https://github.com/debezium/debezium/actions/workflows/release-contributors-list.yml.

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+It's my pleasure to announce the first release of the Debezium 1.7 series, *1.7.0.Beta1*!
+
+This release brings https://docs.nats.io/developing-with-nats-streaming/streaming#nats-streaming-overview[NATS Streaming]
+support for Debezium Server along with many other fixes and enhancements. +
+Also this release is the first one tested with Apache Kafka 2.8.
+
++++<!-- more -->+++
+
+Also this release has a breaking change for the MySQL Connector. +
```
Comment (gunnarmorling) [inline _posts/2021-08-25-debezium-1-7-beta1-released.adoc:17]: Can we add something like this:

Also the Debezium container images for Apache Kafka and Kafka Connect (links tbd.) have been updated to version 2.8. This means that you can test Debezium with the new ZooKeeper-less mode for running Kafka (KIP-500 [link]). We'll share more details in a separate post shortly.

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+The MySQL driver was updated to the latest version 8.0.26 with https://issues.jboss.org/browse/DBZ-3833[DBZ-3833].
+This update comes with a new timezone handling and configuration options. +
```
Comment (gunnarmorling) [inline _posts/2021-08-25-debezium-1-7-beta1-released.adoc:19]: How is the driver update breaking? New options doesn't sound harmful?

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+Detailed information can be found in the https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-connp-props-datetime-types-processing.html[MySQL docs].
+
+This month we added the https://debezium.io/documentation/reference/operations/debezium-ui.html[Debezium UI]
```
Comment (gunnarmorling) [inline _posts/2021-08-25-debezium-1-7-beta1-released.adoc:22]:
```suggestion
Earlier this month, we added the https://debezium.io/documentation/reference/operations/debezium-ui.html[Debezium UI]
```

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc (hunk 6 above)
Comment (gunnarmorling) [inline _posts/2021-08-25-debezium-1-7-beta1-released.adoc:26]: We usually mention a few key issues that were fixed. E.g. see here ("Further Fixes..."): https://debezium.io/blog/2021/06/10/debezium-1-6-beta2-released/. Perhaps something on asorted Oracle connector fixes?

**Review by gunnarmorling — COMMENTED** [review]
@rk3rn3r, a few comments inline. Thanks!

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc (hunk 9 above)
Comment (rk3rn3r) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:19]: according to flags in the Jira and according to @jpechane it is. I think properties for timezone handling changed?

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc (hunk 7 above)
Comment (rk3rn3r) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:27]: Looks like that action is broken:
https://github.com/debezium/debezium/runs/3422085902?check_suite_focus=true

**Comment by rk3rn3r** [comment]
@gunnarmorling  applied most changes. contributors actions seems broken, will go through the list to see if there are issues worth listing (and if I am able to describe those).

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc (hunk 7 above)
Comment (gunnarmorling) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:33]: You need to specify valid tag names ;) (including the leading "v"); I ran it again, see here:

https://github.com/debezium/debezium/runs/3422348517?check_suite_focus=true

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc (hunk 9 above)
Comment (gunnarmorling) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:25]: Ok, add a link to the upgrade procedure for that release (https://debezium.io/releases/1.7/release-notes#release-1.7.0-beta1), which should discuss this (and any other) breaking change.

Comment (rk3rn3r) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:25]: This paragraph has already the same information like that section in the release announcement. The change only affects people that manually provide DSN options or that use the MySQL connector as dependency. All the related information can be taken from the linked DBZ issue and the linked MySQL docs.

Nothing to do for average users that's not mentioned here. Can I leave it how it is?

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc (hunk 6 above)
Comment (rk3rn3r) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc]: I have no idea about any issue that's worth listing here and I have no idea what I could write about the Oracle issues in this release.

**Comment by rk3rn3r** [comment]
I added the contributors list.

File: _posts/2021-08-25-debezium-1-7-beta1-released.adoc (hunk 9 above)
Comment (gunnarmorling) [reply _posts/2021-08-25-debezium-1-7-beta1-released.adoc:25]: Yes, that's ok.

**Comment by gunnarmorling** [comment]
@rk3rn3r, ok, mostly looking good now. Still got a description of some key issues coming up? @Naros may help sketch something on the Oracle ones, if needed.

**Comment by Naros** [comment]
@rk3rn3r Here's a blurb about Oracle in which you can wordsmith as needed:
```
The Debezium connector for Oracle also received several improvements including the ability to configure multiple Oracle RAC nodes with different ports (DBZ-3813), multiple DDL parser fixes (DBZ-3877, DBZ-3893), and improved updating SCN offsets more frequently (DBZ-3876).
```

**Comment by gunnarmorling** [comment]
Thx, @rk3rn3r! Added some things, will merge now.

**Pull request merged**